packages/edit-site/src/components/sidebar-navigation-screen/index.js

```diff
@@ -37,6 +37,7 @@ export default function SidebarNavigationScreen( {
 								__( 'Navigate to the previous view: %s' ),
 								parentTitle
 							) }
+							goToParent
```

Comment:
What's the best way to test changes to this file?

Reply:
You can go to the navigation sidebar in the site editor and try navigating... that said, until #47777 there's only two level  of hierarchy in the tree meaning there's no difference between this branch and trunk so far. We'll have to merge this branch with #47777 to actually see the benefits.